app/controllers/users_controller.rb

```diff
@@ -12,7 +12,9 @@ def index
       mentoring:  'Helping as a Mentor',
       organizing: 'Helping as an Organizer'
     }
-    @users = User.ordered(params[:sort], params[:direction]).group('users.id').with_all_associations_joined
+    @users = User.ordered(params[:sort],
+                          params[:direction]).group('users.id').with_all_associations_joined.
```

Comment:
That's way better! Thanks. 
I take it you do agree with the scope added here inline?